gammapy/modeling/models/spectral.py

```diff
@@ -1440,6 +1450,8 @@ def __init__(
 
         parameters = spectral_model.parameters.parameters.copy()
         parameters.append(par)
+        self.alpha_norm = Parameter("alpha_norm", alpha_norm, frozen=alpha_norm_frozen)
+        parameters.append(self.alpha_norm)
```

Comment:
Use this instead of two append calls:
```
parameters.extend([par, self.alpha_norm])
```
I think this works too and does the same thing, you can use that if you prefer:
```
parameters += [par, self.alpha_norm]
```